Description: Use x-terminal emulator in lieu of xterm for connection edits.
Author: Walter Lapchynski <wxl@ubuntu.com>
Bug: https://launchpad.net/bugs/1786978 
--- a/resources/nm-tray.conf
+++ b/resources/nm-tray.conf
@@ -1,3 +1,3 @@
 [General]
-connectionsEditor=xterm, -e, nmtui-edit
+connectionsEditor=x-terminal-emulator, -e, nmtui-edit
 enableNotifications=true
